src/components/SearchPage/SearchPage.jsx

```diff
@@ -0,0 +1,94 @@
+import { Link } from "react-router-dom";
+import React from "react";
+
+import Footer from "../Footer/Footer";
+import logo from "../../assets/logo.png";
+import "./SearchPage.css";
+import { fetchPokemon } from "../../utils/PokeApi";
+
+function InfoPage() {
+  function handleEnter(event) {
+    if (event.key === "Enter" || event.keyCode === 13) {
+      event.preventDefault();
+      fetchPokemon();
+    }
+  }
+
+  return (
+    <div>
+      <header className="header">
+        <div className="header__logo">
+          <Link to="/" className="header__button">
+            <img className="header__logo-image" src={logo} alt="logo" />
+          </Link>
+          <h1 className="header__logo_text">Search for a Pokemon!</h1>
+        </div>
+        <div className="header__search">
+          <input
```

Comment:
If you have `inputs` then they should be inside a `form` tag to let you add a submit handler